### DISCUSSION TITLE
Game is crashing and blaming mods.

### DISCUSSION THREAD

**Discussion by Dandandooo** [FabricMC/fabric-api#2485] [Player support]
Every time I start the game it goes into an infinite loop of saying that the block states are being requested too early.

Here is the log:
[https://mclo.gs/bx7l8xb](https://github.com/FabricMC/fabric/issues/url)

I tried disabling the specified mod (in this case Paladin Furniture Mod) but then a different mod appeared in the exact location.
Tell me if there is more info that would be helpful.

**Comment by apple502j** [comment]
Please re-install Fabric Loader and Fabric API first; https://fabricmc.net/use/installer/
